### PR TITLE
Fix Bug

### PR DESCRIPTION
**Pull request by VanHieu7119**
Fix Bug
Link redmine: https://edu-redmine.framgia.vn/issues/2716

### CODE REVIEW

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/java/com/framgia/music_39/screen/MainActivity.java [hunk 1]
```diff
@@ -18,12 +18,14 @@
         implements BottomNavigationView.OnNavigationItemSelectedListener {
     private static final int REQUEST_PERMISSION_STORAGE = 123;
     private ActionBar mActionBar;
-    private Navigator mNavigator;
+    public static Navigator mNavigator;
+    private Boolean mPermission;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_39/screen/MainActivity.java]: mPermission -> mIsPermission

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**